### ISSUE TITLE
Comment in SortedList Does Not Match The Code

### ISSUE THREAD

**Issue by taoyouh** [dotnet/runtime#39526]
The comment of EnsureCapacity does not match the code. It says "if the current capacity is less than min" but the code will double the capacity even if min is 0.

https://github.com/dotnet/runtime/blob/c5d60890e37138bbc66dc393b7ee052a3da78763/src/libraries/System.Collections/src/System/Collections/Generic/SortedList.cs#L529

**Comment by danmosemsft** [comment]
I think you're right @taoyouh . Do you want to offer a PR to fix it?

**Comment by taoyouh** [comment]
I'd love to provide a PR.

My question is, shall I fix the code or change the comment? As this is a private method and is called only when resizing is required, I think the code behaviour is acceptable.

**Comment by taoyouh** [comment]
I have submitted my PR.

Further more, I found that the only usage of the method EnsureCapacity is the following line:
https://github.com/dotnet/runtime/blob/c5d60890e37138bbc66dc393b7ee052a3da78763/src/libraries/System.Collections/src/System/Collections/Generic/SortedList.cs#L668

I think this method does not need to check whether "newCapacity < min". If DefaultCapacity is not 0, newCapacity will always be greater than or equal to min.
